sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalogSuite.scala

```diff
@@ -1316,8 +1316,10 @@ abstract class SessionCatalogSuite extends AnalysisTest {
       val funcMeta2 = newFunc("yes_me", None)
       val tempFunc1 = (e: Seq[Expression]) => e.head
       val tempFunc2 = (e: Seq[Expression]) => e.last
-      catalog.createFunction(newFunc("func2", Some("db2")), ignoreIfExists = false)
-      catalog.createFunction(newFunc("not_me", Some("db2")), ignoreIfExists = false)
```

Comment:
Sorry about it , I mistake it when i resolving the conflicts. I'll revert this later.